Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 1000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 5000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4000.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxRenderRadius, "Detail clipmap max render radius", "Max render radius that the lowest resolution clipmap can cover.")
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 50.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 100.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 1000.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxResolution, "Macro clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 10.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution, "Detail clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 2048.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2048.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapScaleBase, "Macro clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapScaleBase, "Detail clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
```

Comment:
Soft min/max seems appropriate, but I'd make the hard min / max lower / higher here - it just needs to prevent breaking behavior.